src/com/sun/ts/tests/jaxrs/spec/client/typedentitieswithxmlbinding/build.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+
+    Copyright (c) 2012, 2020 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project name="jaxrs_spec_client_typedentitieswithjaxb" basedir="." default="usage">
+
+	<property name="app.name" value="jaxrs_spec_client_typedentitieswithjaxb" />
+
+	<property name="resource.classes" value="com/sun/ts/tests/jaxrs/spec/client/typedentitieswithjaxb/Resource.class,
+		com/sun/ts/tests/jaxrs/spec/client/typedentitieswithjaxb/EntityMessageWriter.class,
+		com/sun/ts/tests/jaxrs/spec/client/typedentitieswithjaxb/EntityMessageReader.class,
+		com/sun/ts/tests/jaxrs/ee/rs/ext/messagebodyreaderwriter/ReadableWritableEntity.class" />
+	<property name="appconfig.class" value="com/sun/ts/tests/jaxrs/spec/client/typedentitieswithjaxb/TSAppConfig.class" />
```

Comment:
it should be com/sun/ts/tests/jaxrs/spec/client/typedentitieswithxmlbinding right? or change the path to com/sun/ts/tests/jaxrs/spec/client/typedentitieswithjaxb at other places.

Reply:
Fixed in updated commit.